src/datasource.ts

```diff
@@ -541,7 +541,7 @@ export class OpenSearchDatasource extends DataSourceWithBackend<OpenSearchQuery,
             metric =>
               metric.type === 'raw_data' ||
               metric.type === 'raw_document' ||
-              (request.app === CoreApp.Explore && target.queryType === QueryType.Lucene && metric.type === 'logs')
+              (request.app === CoreApp.Explore && target.queryType === QueryType.Lucene && metric.type)
```

Comment:
```suggestion
              (request.app === CoreApp.Explore && target.queryType === QueryType.Lucene)
```

now that we are sending all lucene explore queries to the backend, do we need the check for metric.type here at all? would it ever be undefined, and if it is, would it matter? this is just a minor nit.

Reply:
Was wondering this myself, tried it out locally. Didn't see anything weird.
I'll merge with your suggestion and continue keeping an eye out. Thank yoou!